src/olympia/activity/tests/test_utils.py

```diff
@@ -312,6 +312,27 @@ def test_task_user_doesnt_get_mail(self, send_mail_mock):
         assert self.developer2.email in recipients
         assert self.task_user.email not in recipients
 
+    @mock.patch('olympia.activity.utils.send_mail')
+    def test_ex_reviewer_doesnt_get_mail(self, send_mail_mock):
+        """If a reviewer has now left the team don't email them."""
+        self._create(amo.LOG.REJECT_VERSION, self.reviewer)
+        # Take his joob!
+        GroupUser.objects.get(group=Group.objects.get(name='Addon Reviewers'),
+                              user=self.reviewer).delete()
+
+        action = amo.LOG.DEVELOPER_REPLY_VERSION
+        comments = u'Thïs is á reply'
+        version = self.addon.latest_version
+        log_and_notify(action, comments, self.developer, version)
+
+        logs = ActivityLog.objects.filter(action=action.id)
+        assert len(logs) == 1
+
+        recipients = self._recipients(send_mail_mock)
+        assert len(recipients) == 1
+        assert self.developer2.email in recipients
+        assert self.reviewer.email not in recipients
```

Comment:
The default case is tested in [`test_developer reply`](https://github.com/mozilla/addons-server/pull/3750/files#diff-def054d2261d09e7ad3da859b87dbe02L217) above.  To test that first we'd literally have to repeat all the same code, plus lines that delete the ActivityLog and emails.   Is that what you're suggesting?